lib/engine/game/g_18_usa/entities.rb

```diff
@@ -295,9 +295,9 @@ module Entities
             name: 'P16 - Regional Headquarters',
             value: 60,
             revenue: 0,
-            desc: 'Regional Headquarters may be used to upgrade a green or brown non-metropolis city after phase 5 begins. ' \
-                  'May be placed on any existing normal city. Three of the track segments are optional and can be placed ' \
-                  'pointing toward water or other off map areas.',
+            desc: 'Regional Headquarters may be used to upgrade a green or brown non-metropolis city after phase 5 begins. '\
+                  'The X23 tile may be placed on any existing normal city. Two of the track segments are optional '\
+                  'and can be placed pointing toward water or other off map areas.',
```

Comment:
Why did you change the optional track segments from three to two? Here are the rules (All-aboard edition):

> Comes with the gray Regional Headquarters tile. #e may be used to upgrade a green or
brown non-metropolis city a$er Phase 5 begins.
Three of the track segments are optional and can be placed pointing toward water or other o!
map areas.